Backend/src/database/services/institution.service.ts

```diff
@@ -301,6 +301,13 @@ export default class InstitutionService {
     return await institutionModel.find(keyword).sort(sortBy).exec();
   }
 
+  static async getUsersInstitutions(userId: (ObjectId | string)):
+      Promise<IInstitution[] | null> {
+    return await institutionModel.find({memberIds: {$elemMatch: {$eq: userId}}});
```

Comment:
Should also additionally search in adminIds 
```suggestion
    return await institutionModel.find({
      $or: [
        { memberIds: { $elemMatch: { $eq: userId } } },
        { adminIds: { $elemMatch: { $eq: userId } } },
      ],
    });
```

Reply:
Yes, I did it eventually